### PR TITLE
Add monitor API for Harbor components

### PR DESCRIPTION
**Pull request by yixingjia**
1>AdminServer
2>Jobservice
3>UI

### CODE REVIEW

**Comment by reasonerjt** [comment]
@yixingjia please rebase to resolve the conflict.

File: src/adminserver/handlers/router.go [hunk 1]
```diff
@@ -27,5 +27,6 @@ func newRouter() http.Handler {
 	r.HandleFunc("/api/configurations", api.UpdateCfgs).Methods("PUT")
 	r.HandleFunc("/api/configurations/reset", api.ResetCfgs).Methods("POST")
 	r.HandleFunc("/api/systeminfo/capacity", api.Capacity).Methods("GET")
+	r.HandleFunc("/api/v1/configurations/ping", api.Ping).Methods("GET")
```
Comment (reasonerjt) [inline src/adminserver/handlers/router.go]: let's not add version to a specific API.
And the `ping` shouldn't be under `configuration`

File: src/ui/router.go [hunk 2]
```diff
@@ -60,6 +60,7 @@ func initRouters() {
 	}
 
 	// API
+	beego.Router("/api/v1/ui/ping", &api.SystemInfoAPI{}, "get:Ping")
```
Comment (reasonerjt) [inline src/ui/router.go]: Similarly, how about just set the URL `/api/ping`

**Comment by just4est** [comment]
Hello. I get 404 on /api/v1/configurations/ping . 
How can i get heath status of jobservice ?

**Pull request merged**